src/runtime/bootstrap-custom-element.ts

```diff
@@ -36,6 +43,25 @@ export const proxyCustomElement = (Cstr: any, compactMeta: d.ComponentRuntimeMet
     cmpMeta.$flags$ |= CMP_FLAGS.needsShadowDomShim;
   }
 
+  // TODO(STENCIL-914): this check and `else` block can go away and be replaced by just `BUILD.scoped` once we
+  // default our pseudo-slot behavior
+  if (BUILD.patchPseudoShadowDom && BUILD.scoped) {
+    patchPseudoShadowDom(Cstr.prototype, cmpMeta);
```

Comment:
I'm not sure if we'll need the `cmpMeta` arg passed in here when this behavior becomes default. Right now, it's only used by a few of these patch functions to prevent patching based on some niche condition.